testsuite/features/step_definitions/setup_wizard_steps.rb

```diff
@@ -0,0 +1,12 @@
+When(/^I enter the scc credentials$/) do
```

Comment:
Don't create a new rb file just for that, please, put it in old file.

`I enter the SCC credentials`